content/_layouts/base.njk

```diff
@@ -73,10 +73,12 @@
       Skip to main
     </a>
 
+    {% typogr %}
     {% include "site/alert.njk" %}
     {% include "site/banner.njk" %}
     {% include 'page/base.njk' %}
     {% include 'site/footer.njk' %}
+    {% endtypogr %}
```

Comment:
@mirisuzanne I noticed we had double-typogr'd code all over the place (e.g. `This <span class="caps"><span class="caps">CSS</span></span> Thing`). The main change here is to remove `typogr` from where it was used manually and where it was auto-used any time we used `md` or `mdInline`, and instead wrap the entire site contents in this `{% typogr %}` shortcode. That's what we're doing on Sass Site as well.

I think the functional downside / difference is that before, the `mdInline` filter was _not_ adding `widont` in cases of 5 words or less... How important is that?

Reply:
@mirisuzanne Nudge on this...

Reply:
This approach makes sense. We'll want to check the output. The problem with applying widont on short text is that you can end up with very strange line balancing. We can see if that problem shows up in a deploy preview? I'd be tempted to remove typogr entirely, and switch to `text-wrap: balance` once it's available anywhere. https://cloudfour.com/thinks/responsive-headlines-are-about-to-get-awesome/

Reply:
Yeah, interesting. You can see the output diff here: https://github.com/oddbird/oddleventy-built/compare/positioning-update..typogr

Reply:
Can't tell from that how it actually ends up looking, but yeah - some of those are a bit worrying. There's really no point putting widont in a 2 & 3 word headlines. It'll just lead to overflow, and doesn't really prevent anything.

Reply:
Hm, it shouldn't add anything when there are only 2 words, but it will add to 3-word headings. I don't see a great alternative at this point?

Reply:
I think the alternative is we apply it less often, rather than more often. For example: only on headlines, and only when they have more than 5 words. Widows are a bit annoying, but I think they might be better than making 3-word headlines wrap after the first word?

Reply:
That might be a question for @SondraE or @stacyk though

Reply:
Or we just turn off the `widont` feature?

Reply:
Disabled in https://github.com/oddbird/oddleventy/commit/1dd8a9ce8d580eda5a9bf095b39daa5940dc37f6

Reply:
I'm good with removing widont all together. IMO, it gets in the way more than it helps.